warehouse/README.md

```diff
@@ -3,9 +3,42 @@
 This dbt project is intended to be the source of truth for the cal-itp-data-infra BigQuery warehouse.
 
 ##  Setting up the project on your local machine
-0. [Authorize the gcloud CLI](https://cloud.google.com/sdk/docs/authorizing) so that you can access BigQuery (implied step -1: make sure that you have permissions for BigQuery)
-1. Create a file `~/.dbt/profiles.yml` (i.e., in your home directory inside a `.dbt` subdirectory) that contains the following:
-```
+
+> Note: if you get `Operation not permitted` when attempting to use the terminal,
+> you may need to [fix your terminal permissions](https://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/)
+
+### Install Homebrew (if you haven't)
+1. Follow the installation instructions at https://brew.sh/
+2. Then, `brew install gdal` to install a geospatial library needed later
+
+### Install the Google SDK (if you haven't)
+0. Implied: make sure that you have GCP permissions (i.e. that someone has added you to the GCP project)
+1. Follow the installation instructions at https://cloud.google.com/sdk/docs/install
+   1. Use `tar -xvf <drag file from Finder window to get file name> ~` to unzip into your home directory
```

Comment:
per feedback from @edasmalchi I think we need to be a bit more explicit -- saying how to open Terminal & how to navigate to home directory within Terminal